locales/bn.json

```
@@ -0,0 +1,37 @@
+{ "lang": "Translated by @SubhrajitSain - https://github.com/SubhrajitSain",
+
+  "text_-XONHh": "<span><strong>Encoder</strong>.js\n</span>",
+  "text_KZFk8S": "<span>ইনপুট\n</span>",
+  "text_S-x5go": "<span>ডক্স\n</span>",
+  "text_Vcjrr0": "<span>আউটপুট\n</span>",
+  "text_v-MT8j": "<span>মোড:\n</span>",
+  "static_8hiseu": "JustStudio দ্বারা Encoder.js",
```

studio's name should be with dot at the end - JustStudio.
```suggestion
  "static_8hiseu": "JustStudio. দ্বারা Encoder.js",
```
-----------------
but if that dot is too wrong for bengali language, you can make it like
```suggestion
  "static_8hiseu": "\"JustStudio.\" দ্বারা Encoder.js",
```
or
```suggestion
  "static_8hiseu": "Encoder.js",
```

---

I am making it "/"JustStudio./" দ্বারা Encoder.js।", just wait a moment...
I will also include the common words some other day.